### PR TITLE
Creating documentation site

### PR DESCRIPTION
**Pull request by ductiletoaster**
Still need to flesh it out. Footer links could be more helpful. Would like to add a section to reference all the tools we use underneath to make sure we give credit where it is due and help support our own documentation. Need to also do some file cleanup, create a logo and favicon and just generally make it look a bit better.

Lastly need to complete all the docs and update the localized readmes to simply point to the appropriate section in here.

https://www.pixeloven.com/

### CODE REVIEW

File: docusaurus/website/docusaurus.config.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+          showLastUpdateTime: true,
+        },
+        theme: {
+          customCss: require.resolve('./src/css/custom.css'),
+        },
+      },
+    ],
+  ],
+  themeConfig: {
+    // image: 'img/logo-og.png', // What is this for ?
```
Comment (jimmynotjim) [inline docusaurus/website/docusaurus.config.js]: Remove commented bits of the config

File: docusaurus/website/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "scripts": {
+    "start": "docusaurus start",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
```
Comment (jimmynotjim) [inline docusaurus/website/package.json]: lol, what's swizzle?

Comment (ductiletoaster) [reply docusaurus/website/package.json]: It's some sort of setup where you can eject some hidden boilerplate out and customize it. Then if you don't like it you can just delete it and it goes back to default. Might eventually give it a try

File: docusaurus/website/docusaurus.config.js (hunk 1 above)
Comment (ductiletoaster) [reply docusaurus/website/docusaurus.config.js]: Want to leave all these comments because I want to explore all this more and some of it isn't documented.

**Pull request merged**